### PR TITLE
syndicate lavaland outpost has more turrets

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request

Hopefully this makes the outpost much harder to raid by reducing the amount of blind spots for the turrets (by adding more turrets) and adding turrets _inside_ the outpost.

image

### Why is this change good for the game?

it should be hard to raid because its a fucking syndicate outpost with turrets and the scientists are armed with antimaterial .50 bmg sniper rifles

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
MORE GUN

### What should players be aware of when it comes to the changes your PR is implementing?
GUN = MORE

### What general grouping does this PR fall under? 
GUN MORE

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
NO

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
GUN MORE

# Changelog

:cl:  
tweak: syndicate lavaland changes
rscadd: MORE GUN 
/:cl:

### CODE REVIEW

**Comment by redmoogle** [comment]
inb4 miners just keep using capsules to get across

**Comment by athena413** [comment]
> 
> 
> inb4 miners just keep using capsules to get across

that is why i put turrets inside too
